bin/gi18n-checkup

```diff
@@ -5,14 +5,15 @@ set +e
 export GOPATH=$HOME/go
 export PATH=$PATH:$GOPATH/bin
 
-go get -u github.com/maximilien/i18n4go/gi18n
-OUTPUT=$?
-if [ $OUTPUT -ne 0 ]; then
-  printf "Cannot install latest gi18n tool to verify strings:\n${OUTPUT}"
-  exit 1
-fi
+# This doesn't compile automatically, so fails
+# go get -u github.com/maximilien/i18n4go
+# OUTPUT=$?
+# if [ $OUTPUT -ne 0 ]; then
+#   printf "Cannot install latest gi18n tool to verify strings:\n${OUTPUT}"
+#   exit 1
+# fi
 
-OUTPUT=`gi18n -c checkup`
+OUTPUT=`i18n4go -c checkup`
```

Comment:
Does this mean it needs to be installed separately? Not clear to me if this has anything to do with the other changes #gonewb